crates/sail-plan/src/function/scalar/misc.rs

```diff
@@ -3,11 +3,11 @@ use std::sync::Arc;
 use datafusion::functions::expr_fn;
 use datafusion_common::ScalarValue;
 use datafusion_expr::{expr, lit, when, ExprSchemable, Operator, ScalarUDF};
+use datafusion_spark::function::bitmap::bitmap_count::BitmapCount;
```

Comment:
if we use the exported functions like this:
```rust
use datafusion_spark::function::bitmap::expr_fn as bitmap_fn;
...
("bitmap_count", F::unary(bitmap_fn::bitmap_count)),
```
we can get rid of the codec overrides for them and simplify the logic a bit